docs/env_vars.md

```diff
@@ -0,0 +1,17 @@
+wlroots reads these environment variables
+
+wlroots specific
+----------------
+* *WLR_DRM_DEVICES*: specifies the DRM device instead of auto probing it
+* *WLR_DRM_NO_ATOMIC*: set to 1 to use legacy DRM interface instead of atomic mode setting
+* *WLR_LIBINPUT_NO_DEVICES*: set to 1 to not fail without any inpout devices
+* *WLR_WL_OUTPUTS*: when using the wayland backend specifies the number of outputs
+* *WLR_X11_OUTPUTS*: when using the X11 backend specifies the number of outputs
+
+generic
+-------
+* *DISPLAY*: if set probe X11 backend in *wlr_backend_autocreate*
+* *WAYLAND_DISPLAY*, *_WAYLAND_DISPLAY*, *WAYLAND_SOCKET*: if set probe Wayland backend in *wlr_backend_autocreate*
```

Comment:
I don't think we support `WAYLAND_SOCKET`? (Yes we should)

Reply:
It's locked at in backend/backend.c for autoprobing backends (only if it's set or not)

Reply:
Oh, you're right. It's just that `wlr_wl_backend_create` doesn't support being passed a socket.